jest-common/pom.xml

```diff
@@ -69,6 +69,11 @@
         </dependency>
 
         <dependency>
+            <groupId>org.easytesting</groupId>
```

Comment:
@kramer 
fest assertions works nicely together with JUnit and Hamcrest.
It offers a readable fluent API and meaningful assertion messages (if tests fails).
May I ask why stick with junit and not extending the possibilities of assertions?